projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-form/payment-form.component.spec.ts

```diff
@@ -81,6 +81,14 @@ class MockCardComponent {
   content: any;
 }
 
+@Component({
+  selector: 'cx-icon',
+  template: '',
+})
+export class MockCxIconComponent {
```

Comment:
You need to add this class to the declarations array (at line 139), so declarations will be : 
```
      declarations: [
        PaymentFormComponent,
        MockCardComponent,
        MockBillingAddressFormComponent,
        MockCxIconComponent
      ],
```